packages/core/src/components/Button/__stories__/Button.stories.tsx

```diff
@@ -6,6 +6,7 @@ import { createStoryMetaSettingsDecorator } from "../../../storybook/functions/c
 import Button from "../Button";
 import "./Button.stories.scss";
 import { Meta, StoryObj } from "@storybook/react";
+import Text from "../../Text/Text"
```

Comment:
can you please insert
```suggestion
import Text from "../../Text/Text";
```
the lint in the CI failed for it